src/app/shared/services/common/loader/loader.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { LoadingController } from '@ionic/angular';
+import { SPINNER_TYPE } from './spinner-enums';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoaderService {
+  constructor(public loadingController: LoadingController) {}
+
+  async showLoader(
+    messageValue = 'Loading, Please Wait...',
```

Comment:
Can you make an interface for this and export it ?

Reply:
Done